manim/animation/fading.py

```diff
@@ -16,15 +16,13 @@
 ]
 
 
-from ..animation.animation import Animation
-from ..animation.animation import DEFAULT_ANIMATION_LAG_RATIO
-from ..animation.transform import Transform
+from .. import logger
+from ..animation.animation import (DEFAULT_ANIMATION_LAG_RATIO, Animation,
+                                   Transform)
```

Comment:
This is breaking the tests: Transform comes from animation.transform, not animation.animation